ls1intum/Artemis
# Bugfix: Fix Sentry issues

### Checklist

### Motivation and Context
Remove errors from Sentry.

### Steps for Testing
- Go to the `Participation` page for any exercise and click on the linked participant name and it should route to the participant detail view.
![image](https://user-images.githubusercontent.com/33764166/66724017-11fa1400-ee10-11e9-8c38-8f01a061077f.png)
- Start any exercise and see if it works
